### PR TITLE
revert: feature flag for guided onboarding copy

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Reverts #2817 due to:
- The onboarding text was displayed in non onboarding pin displays.
- Empty Scroll view was causing unexpected shifts to pincode entry layout.

#### Screenshots Before Revert / After Revert
<p align="center">

</p>

### Other changes

N/A

### Tested

Tested locally on iOS

### How others should test

Devs: Please double check my changes to the migrations.

QA: Check that Pin Entry Functions as expected and displays correctly during onboarding, viewing recovery phrase, and on Pin Change.

### Related issues

N/A

### Backwards compatibility

Yes - Devs may need a fresh install due to redux changes.

### CODE REVIEW

File: src/pincode/Pincode.tsx [hunk 1]
```diff
@@ -64,14 +58,6 @@ function Pincode({
       <View style={styles.spacer} />
       {!errorText && <Text style={styles.title}>{title || ' '}</Text>}
       {!!errorText && <Text style={styles.error}>{errorText}</Text>}
-      <ScrollView contentContainerStyle={styles.contentContainer}>
-        {showGuidedOnboarding && (
```
Comment (cajubelt) [inline src/pincode/Pincode.tsx:68]: wouldn't it be easier to just lift this `{showGuidedOnboarding && ...` part up one level? So turning the feature flag off hides the whole scroll view. Then we wouldn't need to mess with the migrations

**Review by cajubelt — APPROVED** [review]
this is fine, but I pointed out what looks to me like an easier and maybe safer way to do this.

File: src/pincode/Pincode.tsx (hunk 1 above)
Comment (MuckT) [reply src/pincode/Pincode.tsx:68]: @cajubelt I am inclined to revert this then reevaluate the fixed version of https://github.com/valora-inc/wallet/pull/2817 when it is ready.

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest Android internal release build and Latest iOS TestFlight release build V 1.40.0 and updated status in Test rails

**Observation :** Pin page is appearing on recovery phrase, settings, create account flow , restore account flow properly

**Test Rail links:**
**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/69540
**Android 11:** https://valoraapp.testrail.io/index.php?/tests/view/69571

**Devices:**  Google Pixel 2XL(11.0) , iPhone 12(14.7.1)

Thanks.!

**Pull request merged**